### PR TITLE
Update Cookiecutter to 2.1.1

### PR DESCRIPTION
**Pull request by stijnvandrunen**
Since all the CI tests are Python 3.7 anyways I would like Cookiecutter to be updated to the latest version (would love to use the private variables and local template extensions).

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -7,8 +7,8 @@ license = "MIT"
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "^3.6.2"
-cookiecutter = "^1.6"
+python = "^3.7"
+cookiecutter = "2.1.1"
```
Comment (samj1912) [inline pyproject.toml:11]:
```suggestion
cookiecutter = ">=1.7"
```

**Comment by stijnvandrunen** [comment]
Thanks for the edit! Nice change, how would I get this reviewed and merged?

**Comment by stijnvandrunen** [comment]
Thanks for the edit! Nice change, how would I get this reviewed and merged?

**Comment by stijnvandrunen** [comment]
Btw implements #162

**Review by samj1912 — APPROVED** [review]

**Pull request merged**